src/server/helpers/middleware.js

```diff
@@ -171,15 +171,14 @@ export function makeCollectionLoader() {
 		if (commonUtils.isValidBBID(collectionId)) {
 			try {
 				const collection = await new UserCollection({id: collectionId}).fetch({
-					require: false,
-					withRelated: ['collaborators.collaboratorDetail']
+					require: true,
+					withRelated: ['collaborators.collaboratorDetails']
```

Comment:
Looking at naming for other models, I'd expect 'collaborators.details' here

Reply:
https://github.com/bookbrainz/bookbrainz-data-js/blob/fc02949c3426a7eb213ad68df6c544045709fbf9/src/models/userCollectionCollaborator.js#L24
I think `UserCollection.collaborators.collaboratorDetails` is better because there is a `editor.collaboratesOn.collectionDetails` as well

Or Maybe `editor.collaboratesOn.collection` and `UserCollection.collaborators.details` sounds better..
What do you think?

Reply:
I would suggest `UserCollectionCollaborator.collaborator()` and `UserCollectionCollaborator.collection()`

Looking at `Editor.collaboratesOn.collection`, you could skip a step: with the ORM, you can return the array of `collection` models an editor collaborates on directly, with `[.through()`](https://bookshelfjs.org/api.html#Collection-instance-through).

So you would have instead `editor.collaboratesOn()` which would return a collection of UserCollection items.
Same thing for `UserCollection.collaborators()` which would return a collection of Editor items.
Here's an example where this is used in the ORM: https://github.com/bookbrainz/bookbrainz-data-js/blob/fc02949c3426a7eb213ad68df6c544045709fbf9/src/models/editor.js#L25